src/EmailCardList/EmailCard/EmailCard.js

```diff
@@ -0,0 +1,15 @@
+const EmailCard = (props) => {
+    return (
+        <div className={(props.read === true ? 'btn btn-light col-md-12 border-bottom' : 'btn btn-secondary border-bottom col-md-12') + " d-flex flex-row justify-content-between p-2"}>
+            <div className="emailContainer">
+                <p className={'font-weight-bold h5 text-left'}>{props.name}</p>
```

Comment:
it's not great to have a p tag styled as an h5, it would be better to either use the respective semantic heading tag e.g. h3 or to just style the p tag to be big using other means